Features/MultiSelect/user_inputs/multiselect_user_inputs.py

```diff
@@ -0,0 +1,42 @@
+""""Contains test data that are used as user inputs across various areass used in Case Serach"""
+
+
+class MultiSelectUserInput:
+    """User Test Data"""
+
+    """App Names"""
+    multiselect_app_name = "Multi-Select"
+
+    """Menu Names"""
```

Comment:
Just some cosmetic changes. Name the variables starting with lower case.

Reply:
https://github.com/dimagi/dimagi-qa/pull/209/commits/46bed4f851a154e7e97c440d84d72572c1ef173e